src/components/DiagramsLow.vue

```diff
@@ -17,24 +17,24 @@
     <!-- FIGURE CAPTION -->
     <template v-slot:figureCaption>
       <p>
-        Click each diagram to enlarge it and learn more.
+        Click to enlarge.
       </p>
     </template>
     <!-- EXPLANATION -->
     <template v-slot:belowExplanation>      
       <p>
-        Studying all these dynamics across multiple ecosystems in the west is a huge task, but even that list isn’t complete.  In addition to soil saturation, SWE depth, and melt timing, there are lots of other factors that impact how snow turns into flow, such as:  
+        Quantifying snow dynamics across multiple ecosystems in the west is a huge undertaking. In addition to soil saturation, SWE depth, and melt timing, there are numerous other factors that affect how snow turns into streamflow, such as:  
       </p>
       <Sidebar>
         <template v-slot:sidebarTitle>
           Sublimation
         </template>
         <template v-slot:sidebarMessage>
           <p>
-            Some snow never creates meltwater because it evaporates directly back into the atmosphere in a process called <a
+            Some snow evaporates directly back into the atmosphere in a process called <a
               href="https://www.usgs.gov/centers/co-water/science/snowpack-sublimation-measurements-and-modeling-colorado-river-basin?qt-science_center_objects=0#qt-science_center_objects"
               target="_blank"
-            >sublimation</a>. The longer snow is exposed to weather that is sunny but still cold, the more snow vaporizes directly back into the atmosphere<sup>11</sup>.
+            >sublimation</a>. Cold sunny weather contributes to the amount of snow vaporized directly back into the atmosphere<sup>11</sup> and higher wind speeds can accelerate sublimination.
```

Comment:
Issue #102 IPDS reviewer DS wants to add this line: 

> "Higher wind speeds can accelerate sublimation, and sublimation losses are much more significant in the more continental areas (Colorado, Wyoming) and less significant in the more coastal mountains."

I'm okay adding it without a citation, since we do link the USGS.gov snowpack modeling page which links to all of the science. If you're okay with it too, would you add that in?

Reply:
I added it to the end of the sentence in a shortened way.